tests/model_serving/model_server/storage/pvc/utils.py

```diff
@@ -10,12 +12,29 @@ def create_isvc(
     name: str,
     namespace: str,
     deployment_mode: str,
-    storage_uri: str,
     model_format: str,
     runtime: str,
+    storage_uri: Optional[str] = None,
+    storage_key: Optional[str] = None,
+    storage_path: Optional[str] = None,
     min_replicas: int = 1,
     wait: bool = True,
-) -> InferenceService:
+) -> Generator[InferenceService, Any, Any]:
+    predictor_storage = {
+        "minReplicas": min_replicas,
+        "model": {
+            "modelFormat": {"name": model_format},
+            "version": "1",
+            "runtime": runtime,
+        },
+    }
+    if storage_uri:
+        predictor_storage["model"]["storageUri"] = storage_uri  # type: ignore
```

Comment:
why is the `type:ignore` needed?
will it be solved if typing is added to `predictor_storage`?

Reply:
Without the ignore I'd get a complaint from mypy, `Unsupported target for indexed assignment ("object")`